modules/deploying-models-on-the-single-model-serving-platform.adoc

```diff
@@ -89,6 +89,9 @@ The *Deploy model* dialog opens.
 . From the *Model framework (name - version)* list, select a value.
 . In the *Number of model server replicas to deploy* field, specify a value.
 . From the *Model server size* list, select a value.
+. The following options are available only if you have enabled GPU support on your clutser and created an accelerator profile:
```

Comment:
```suggestion
. The following options are only available if you have enabled GPU support on your cluster and created an accelerator profile:
```
the available only vs only available is slightly nitpicky - I can't remember the exact reason why my brain wants "only available" to back up my assertion so feel free to disregard!